examples/benchmarking.ipynb

```diff
@@ -0,0 +1,180 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Welcome to ProgPy's Benchmarking Example"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The goal of this notebook is to demonstrate the Benchmarking feature offered for Prognostic Models. Specifically, we will demonstrate via a simple example how to benchmark the computational efficiency of models!"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, we need to import the necessary modules."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from prog_models.models import BatteryCircuit\n",
+    "from timeit import timeit"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The first import is importing ProgPy's BatteryCircuit Model, and the second import will be used to benchmark our model!"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, let's initialize our Battery Circuit by creating model object."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Step 1: Create a model object\n",
+    "batt = BatteryCircuit()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Then, for our model, we would need to define a future loading function. More information on what a future loading function is and how to use it can be found here: https://nasa.github.io/progpy/prog_models_guide.html#future-loading"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Step 2: Define future loading function \n",
+    "loading = batt.InputContainer({'i': 2})  # Constant loading\n",
+    "def future_loading(t, x=None):\n",
+    "    # Constant Loading\n",
+    "    return loading"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Since this is a simple example, we are going to have a constant loading!"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally, we are going to need to benchmark the simulation!\n",
+    "\n",
+    "We can do this by using the `timeit()` function and pass in our `simulate_to()` or `simulate_to_threshold()` function for the `stmt` argument. For more information regarding the `timeit()` function, please read it's documentation located here: https://docs.python.org/3/library/timeit.html"
```

Comment:
```suggestion
    "We can do this by using the `timeit()` function and pass in our `simulate_to()` or `simulate_to_threshold()` function for the `stmt` argument. For more information regarding the `timeit()` function, please read its documentation located here: https://docs.python.org/3/library/timeit.html"
```

Reply:
Would there be another way to write: "function for the 'stmt' argument" that would convey the appropriate information? I didn't know what this meant. It's clear once I looked up timeit() (and maybe is clear to more code-oriented people), but just a suggestion to maybe reword for accessibility to all users.